pydash/numerical.py

```diff
@@ -383,6 +385,56 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, op):
+    """Return the result of the math operation on the given values."""
+    # Note: If the op is division or multiplication, we need to assign 1 as
+    # default value to the values(needs to be passed in from the outer
+    # methods).
+    try:
+        if not pyd.is_number(value1):
+            value1 = float(value1)
+        if not pyd.is_number(value2):
+            value2 = float(value2)
+    except Exception:
+        pass
+
+    try:
+        return op(value1, value2)
+    except ZeroDivisionError:
+        # The user can pass in the value2 as 0. When the op is division, then
+        # we return INFINITY.
+        return INFINITY
+
+
+def subtract(minuend, subtrahend):
+    """Subtracts two numbers.
+
+    Args:
+        minuend (int/float): Value passed in by the user.
+        subtrahend (int/float): Value passed in by the user.
+
+    Returns:
+        int/float: Result of the difference from the given values.
+
+    Example:
+
+        >>> subtract(10, 5)
+        5
+        >>> subtract(-10, 4)
+        -14
+        >>> subtract(2, 0.5)
+        1.5
+
+    .. versionadded:: TODO
+    """
+    # If the user has given any Nonetype values, we assign 0 as the default
+    # value to the values.
+    minuend = 0 if not minuend else minuend
+    subtrahend = 0 if not subtrahend else subtrahend
```

Comment:
If we decide to change how defaults are set, it would be better to only have a single place that sets the defaults vs. setting the default in every math operation function and having the logic implement in all of those places.